src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,21 @@
-export const TodoInfo = () => {};
+import React from 'react';
+import cn from 'classnames';
+import { TodoType } from '../../types/TodoType';
+import { UserInfo } from '../UserInfo';
+
+type Props = {
+  todo: TodoType;
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  return (
+    <article
+      data-id={todo.id}
+      className={`TodoInfo ${cn({ 'TodoInfo--completed': todo.completed })}`}
+    >
+      <h2 className="TodoInfo__title">{todo.title}</h2>
+
+      <UserInfo user={todo.user} />
```

Comment:
You can check if user exists 
```
{todo.user && <UserInfo user={user} />}
```
and than in UserInfo you can write code differently like that  
```
type Props = {
  user: UserType;
};

export const UserInfo: React.FC<Props> = ({ user }) => {
  return (
    <a className="UserInfo" href={`mailto:${user.email}`}>
      {user.name}
    </a>
  );
};

Reply:
@ViktoriiaRepo  good point